### ISSUE TITLE
QR not displaying

### ISSUE THREAD

**Issue by bstnsolari** [grokability/snipe-it#11778]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Hello everyone.

I'm aware that there's a lot of Bug Reports due to QR code not being displayed, and the main solution is due to the APP_URL in the .env file.

I have Snipe-IT set up on CPanel. Working perfectly.

Snipe-IT version: v6.0.10

.env file configuration:

![env file](https://user-images.githubusercontent.com/94856416/188197702-dd54a2a8-3f02-4c66-9c68-5f97609dcfd1.png)

I also check the option "display qr code" on Barcode settings

When the page loads (public/hardware/assetid) this is what the console returns (Firefox Browser)

![image (2)](https://user-images.githubusercontent.com/94856416/188194838-1cd7fcaf-3b1b-41f9-9dc9-c0e636352dc4.png)

And if I click the Generate Label (or generate barcode... I have Snipe-IT on spanish) this is what the console returns (Firefox Browser)

![Uploading image (1).png…]()

So basically, it's adding "hardware/localhost" to the QR code url. If I remove that, I can visualize the QR code. The route ends up in: "https://equipos.fgs.cl/public/hardware/localhost/hardware/1/qr_code"

APP _DEBUG on true displays this error:

![image](https://user-images.githubusercontent.com/94856416/188195549-2b2375f7-1ee7-4751-9750-c96ee07e136c.png)

### Reproduction steps

1. Check Display QR code on Barcode settings
2.
3.
...

### Expected behavior

Display the QR code of the asset.

### Snipe-IT Version

 v6.0.10

### Operating System

Linux

### Web Server

cPanel

### PHP Version

7.4.29

**Comment by bstnsolari** [comment]
Nevermind. Solved the issue. It was, as all the previos QR Issues solved, due to API_URL on the .env file. The API_URL had to be, for example: https://gmail.com/public _**instead of**_ localhost
